pmm-tests/ps-specific-tests.bats

```diff
@@ -176,3 +187,16 @@ echo "$output"
 	done
 
 }
+
+@test "run pmm-admin purge mysql:metrics" {
```

Comment:
I guess this test should be named like "run pmm-admin purge mysql" not mysql:metrics

Reply:
No that is the correct way I think, because we have to provide specific service name, while asking the client to purge the data 
```
pmm-admin purge mysql -> return this
Error purging mysql data : bad service type.

Service type takes the following values: linux:metrics, mysql:metrics, mongodb:metrics, proxysql:metrics.